src/types/form.ts

```diff
@@ -39,20 +39,20 @@ import { FormLogicSchema, LogicDto } from './form_logic'
 import { IPopulatedUser, IUserSchema, PublicUser } from './user'
 
 export {
```

Comment:
hmm I thought we were going to clean up this importing and re-exporting? can't everything just import from the same place? I feel like this complicates our import/export dependencies

Reply:
I think we should still have a central backend types that reexports shared typed, cause sometimes backend will have their own overrides of shared types, and its cleaner to have all type imports from backend instead of some from shared and some from the backend type folder?

actually kind of a moot point though... since we have instances of importing from both shared and types T_T.

I'll change it to import directly from shared types then in that case if there is no need to override types in the backend

Reply:
DONE